packages/odyssey-react-mui/src/labs/SideNav/SideNav.tsx

```diff
@@ -267,10 +267,14 @@ const SideNav = ({
         },
       );
     }
+
     if (intersectionObserverRef.current && scrollableContentRef.current) {
-      const ul = scrollableContentRef.current;
-      const li = ul?.firstChild;
-      intersectionObserverRef.current.observe(li as HTMLElement);
+      const ulElement = scrollableContentRef.current;
+      const [liElement] = Array.from(ulElement?.children || []);
+
+      if (liElement) {
+        intersectionObserverRef.current.observe(liElement);
+      }
```

Comment:
The `as` wasn't being true to TypeScript and would cause an error on the initial load of UI Shell which has no elements.